src/test/resources/features/refapp-2.x/06-user/users.feature

```diff
@@ -1 +1,30 @@
-Feature: Users Management
\ No newline at end of file
+Feature: User Account Management
+
+  Background:
+    Given a user clicks on system administartion app from home page
+    Then the system loads system administration page
+    When a user clicks on manage accounts app
+    Then the system loads manage acccount page
+    When a user clicks add new account button
```

Comment:
Based on this [info](https://johnfergusonsmart.com/1136-2/) @sherrif10  shared, am tempted to change to And for step line 8